common/app/views/fragments/items/elements/facia_cards/image.scala.html

```diff
@@ -12,6 +12,7 @@
     maybeImageMedia: Option[ImageMedia] = None,
     maybePath: Option[String] = None,
     maybeSrc: Option[String] = None,
+        lazyLoadIndex: Boolean = false,
```

Comment:
I think now we've changed to a boolean here we should rename all the variables so instead of `index` or `containerIndex` we should call it `shouldLazyLoad`